exercises/anagram/src/test/java/AnagramTest.java

```diff
@@ -52,6 +67,20 @@ public void testIdenticalWordIsNotAnagram() {
 
     @Ignore
     @Test
+    public void testCapitalWordIsNotAnagram() {
```

Comment:
I think naming this `testCapitalWordIsNotOwnAnagram ` would match the canonical test description ("capital word is not own anagram") slightly better.